cpp/src/aztec3/circuits/rollup/base/nullifier_tree_testing_harness.cpp

```diff
@@ -10,13 +11,74 @@ NullifierMemoryTreeTestingHarness::NullifierMemoryTreeTestingHarness(size_t dept
     : NullifierMemoryTree(depth)
 {}
 
+// Check for a larger value in an array
+bool check_has_lesser_value(std::vector<fr> const& values, fr const& value)
+{
+    // Must perform comparisons on integers
+    uint256_t value_as_uint = uint256_t(value);
+    for (auto const& v : values) {
+        if (uint256_t(v) < value_as_uint) {
+            return true;
+        }
+    }
+    return false;
+}
+
+// TODO: test
```

Comment:
note: this is going to change with https://github.com/AztecProtocol/aztec3-circuits/issues/161 hence why its not deeply integrated yet. The purpose of this fix is to unblock pacakges